.changes/unreleased/tinymce-TINY-10581-2024-01-24.yaml

```
@@ -0,0 +1,6 @@
+project: tinymce
+kind: Fixed
+body: In some cases having a vertical scroll made the toolbar width calculation wrong.
```

It is not clear what exactly the vertical scroll is for. It sounds to me like the vertical scroll belongs to the toolbar itself. 
```suggestion
body: The toolbar width was miscalculated for the inline editor positioned inside a scrollable container.
```

---

changed